src/fragments/forms/place-input/i18n/place-input.i18n.cs-cz.js

```diff
@@ -0,0 +1,34 @@
+
+export default {
+  placeInput: {
+    findAPlace: 'Najít místo',
+    place: 'Místo',
+    addRouteStop: 'Přidat zastávku na trase',
+    routePlace: 'Zastávka na trase',
+    routeDestination: 'Cíl',
+    placeNameRequired: 'Název místa je vyžadován',
+    yourLocation: 'Vaše poloha',
+    couldNotDetermineYourLocation: 'Vaši polohu se nepodařilo zjistit. Ujistěte se, že jste získali požadovaná oprávnění',
+    useYourLocation: 'Použijte svou aktuální polohu',
+    removeInput: 'Odstranit místo/vstup',
+    directions: 'Instrukce',
+    fromYourBrowser: 'Z vašeho zařízení',
+    addStartingPlace: 'Přidání výchozího místa',
+    startingPlace: 'Výchozí místo',
+    openInputOptions: 'Otevřít možnosti vstupu',
+    addInput: 'Přidat místo vstupu',
+    switchCoords: 'Přepnout souřadnice',
+    rawCoordinate: 'Lat,lng hrubé souřadnice',
+    fillOrRemoveInput: 'Vyhledejte místo nebo použijte tlačítko na mapě vpravo',
+    noPlaceFound: 'Nebyly vráceny žádné výsledky',
+    unknownSearchPlaceError: 'Chyba hledání neznámého místa',
+    approximateDistance: 'Přibližná vzdálenost po přímce od aktuálního středu mapy',
+    couldNotDetermineYourPreciseLocationUsingPreviousLocation: 'Vaši přesnou polohu nebylo možné určit. Místo toho bude použita vaše předchozí poloha',
+    pleaseTypeSomething: 'Zadejte něco k vyhledávání',
```

Comment:
Automatic translation suggests "specify something" which differs from the original "please type something", prompting the user to the concrete typing action. Do you think this is enough in this context, not telling the user how to do it?